### PR TITLE
Input parameter to decide whether or not to perform the JIRA server info check

### PR DESCRIPTION
**Pull request by gtaroni**
This is addressing https://github.com/DeloitteDigitalUK/jira-agile-metrics/issues/14

The JIRA server may return 403 while creating the JIRA object. This may due to security restrictions on the server. I recreated the issue, see logs below:

[2019-12-19 17:07:14 DEBUG] Starting new HTTPS connection (1): my.jira.com:443
[2019-12-19 17:07:14 DEBUG] https://my.jira.com:443 "GET /rest/api/2/serverInfo HTTP/1.1" 403 None

The new flag jira-server-version-check will skip the serverInfo check. By default, this check is always performed

### CODE REVIEW

File: jira_agile_metrics/cli.py [hunk 1]
```diff
@@ -17,7 +17,7 @@ def configure_argument_parser():
     """Configure an ArgumentParser that manages command line options.
     """
 
-    parser = argparse.ArgumentParser(description='Extract Agile metrics data from JIRA and produce data and charts.')
+    parser = argparse.ArgumentParser(description='Extracting Agile metrics data from JIRA and produce data and charts.')
```
Comment (optilude) [inline jira_agile_metrics/cli.py]: Why did you change this? The original is grammatically correct (and the change is not). Do you mind changing it back in this PR?

File: jira_agile_metrics/cli.py [hunk 2]
```diff
@@ -114,6 +115,8 @@ def get_jira_client(connection):
     password = connection['password']
     http_proxy = connection['http_proxy']
     https_proxy = connection['https_proxy']
+    jira_server_version_check = connection['jira_server_version_check']
+    logger.debug("Cheking JIRA server version: %s", jira_server_version_check)
```
Comment (optilude) [inline jira_agile_metrics/cli.py]: Can we make this defaults to true if that was the old behaviour? Can you also remove the debug logging here?

File: jira_agile_metrics/cli.py (hunk 1 above)
Comment (gtaroni) [reply jira_agile_metrics/cli.py]: Indeed, an oversight, I will fix it

File: jira_agile_metrics/cli.py (hunk 2 above)
Comment (gtaroni) [reply jira_agile_metrics/cli.py]: The default value is set to true in config.py
I am removing the debug line.

**Comment by gtaroni** [comment]
Thanks for your input, I am taking on your feedback and submit a new PR.

**Comment by gtaroni** [comment]
I resubmitted a new PR with the requested modifications. Thanks for the feedback
